CHANGELOG.md

```diff
@@ -6,6 +6,24 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### BREAKING CHANGES
+
+- **CUMULUS-2099**
+  - `meta.queues` has been removed from Cumulus messages.
```

Comment:
I wrote it this way because while I only removed it from the message template - the net effect is that it will be missing from all Cumulus workflow messages, so no should rely on it anymore. I'm not sure of the distinction between Cumulus workflow messages and Cumulus "core" workflow messages? Is it possible to have a workflow message that isn't based on the message template?

Reply:
So the Cumulus Message is a 'format' that is expected by external tooling (e.g. CMA) whereas Core's >message< is a specific implementation of that.  

We may not want to muddle this too much, but should probably make sure we make our language consistent, or at least more so, particularly if we have a component Core setup.  

That said, this is a nit, take/leave and this can be punted to an arch meeting rather than hold up this PR.

Reply:
I'm happy to just update the language for clarity/consistency, just didn't understand the distinction

Reply:
ca1a3cce03c9f96506b943ee770440e1322db228

Reply:
This seems good for this PR.  I think we need to address doc language more wholesale, but that does not have to happen here.